atlasreader/data/README.md

```diff
@@ -0,0 +1,337 @@
+- TEJ Behrens, H. Johansen-Berg, MW Woolrich, SM Smith, CAM Wheeler-Kingshott, PA Boulby, GJ Barker, EL Sillery, K. Sheehan, O. Ciccarelli , AJ Thompson, JM Brady, PM Matthews (2003). Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nature Neuroscience, 6, 750–757.
+- Jörn Diedrichsen, Joshua H Balsters, Jonathan Flavell, Emma Cussans, Narender Ramnani (2009). A probabilistic MR atlas of the human cerebellum. Neuroimage, 46, 39-46.
+
+### License
+
+The FSL templates and atlases are under the License that can be found
+[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Licence).
+
+## ICBM 2009c Nonlinear Asymmetric
+
+A number of unbiased non-linear averages of the MNI152 database have been
+generated that combines the attractions of both high-spatial resolution and
+signal-to-noise while not being subject to the vagaries of any single brain
+(Fonov et al., 2011). The procedure involved multiple iterations of a process
+where, at each iteration, individual native MRIs were non-linearly fitted to
+the average template from the previous iteration, beginning with the MNI152
+linear template. The templates present an unbiased standard magnetic resonance
+imaging template brain volume for normal population. These volumes were created
+using data from ICBM project. For more information see: http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009.
+
+### Creation of template
+
+The template was download on 24. Sept. 2018 from
+http://www.bic.mni.mcgill.ca/~vfonov/icbm/2009/mni_icbm152_nlin_asym_09c_nifti.zip.
+We used the following command to create a T1 brain only template:
+
+```
+fslmaths mni_icbm152_t1_tal_nlin_asym_09c.nii \
+    -mul mni_icbm152_t1_tal_nlin_asym_09c_mask.nii \
+    mni_icbm152_t1_tal_nlin_asym_09c_brain.nii.gz \
+    -odt short
+```
+
+### License
+
+Copyright (C) 1993-2004 Louis Collins, McConnell Brain Imaging Centre, Montreal
+Neurological Institute, McGill University. Permission to use, copy, modify, and
+distribute this software and its documentation for any purpose and without fee
+is hereby granted, provided that the above copyright notice appear in all
+copies. The authors and McGill University make no representations about the
+suitability of this software for any purpose.  It is provided "as is" without
+express or implied warranty.  The authors are not responsible for any data loss,
+equipment damage, property loss, or injury to subjects or patients resulting
+from the use or misuse of this software package.
+
+# Atlas Information
+
+`atlasreader` contains many different atlases that each are under their own
+license and related to specific publications. This README file acknoledges those
+licenses, describes the origin of the atlases and how they were acquired and
+explains how the atlases were adapted to fit into the `atlasreader` framework.
+Many of those atlases were downloaded with `nilearn.datasets`'s `fetch_`
```

Comment:
Link to nilearn docs?

Reply:
You're right! https://github.com/miykael/atlasreader/pull/67/commits/f11d073e29ecfd08e24dec95e5e046d4f4de1811 takes care of this.